spec/views/exports/hamilton_lrc_pbmc_pools_to_cellaca_count.csv.erb_spec.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'exports/hamilton_lrc_pbmc_pools_to_cellaca_count.csv.erb' do
+  has_a_working_api
+
+  let(:wells) do
+    # 8 because of the Pools plate.
+    (1..8).map do |index|  # one-based index
+      aliquots = []
+
+      # Initiate 10 aliquots per each well
+      (1..10).each do |_i|
+        supplier_name = "vac-tube-barcode-#{index}"
+        sample_metadata = create(:v2_sample_metadata, supplier_name: supplier_name)
+        sample = create(:v2_sample, sample_metadata: sample_metadata)
+        aliquots << create(:v2_aliquot, sample: sample)
+      end
+      location = WellHelpers.well_at_column_index(index - 1)
+      create(:v2_well, aliquots: aliquots, location: location)
+    end
+  end
+
+  let(:plate) do
+    # Empty wells will be filtered out.
+    wells[0].aliquots = []
+    wells[1].aliquots = []
+
+    # Failed wells will be filtered out.
+    wells[2].state = 'failed'
+    wells[3].state = 'failed'
+
+    create(:v2_plate, wells: wells)
+  end
+
+  before do
+    Settings.purposes = {
+      plate.purpose.uuid => {
+        presenter_class: {
+          args: {
+            required_number_of_cells: 30_000,
```

Comment:
Can you just put the values for these 3 constants in lets at the top of the test? Used in more than one place and they may change in future.